app/src/modules/iot_pollution/iot_pollution.js

```diff
@@ -0,0 +1,235 @@
+                      "@_date": "String",
+                      "@_geo_loc": {
+                        "@_lat": "number",
+                        "@_lon": "number",
+                      },
+                    },
+                    "@_headers": "object",
+                    "meta": {
+                      "active": "boolean",
+                      "author": "String",
+                      "updater": "String",
+                      "updatedAt": "String",
+                      "deletedAt": "String",
+                      "createdAt": "number",
+                    }
+                  }
+                ],
+                "@_aggregations": {
+                  "NO2": {
+                    "value": "number",
+                  },
+                  "O3": {
+                      "value": "number",
+                  },
+                  "SO2": {
+                      "value": "number",
+                  },
+                  "PM2.5": {
+                      "value": "number",
+                  },
+                  "PM10": {
+                      "value": "number",
+                  }
+                }
+              };
+              // Call the API and get back data
+              apiCaller.call(api_request, structure, proxyURL, this.timeout, redisTools).then((apiRes) => {
+              console.log(apiRes);
+              Object.keys(apiRes.aggregations).map(aggregation => {
+            		return apiRes.aggregations[aggregation].value = apiRes.aggregations[aggregation].value ? Number.parseFloat(apiRes.aggregations[aggregation].value).toFixed(2) : null;
+            	});
+            	apiRes.result.forEach(function(data) {
+            		data.content.date = moment(data.content.date).locale(language).format('dddd Do MMM YYYY');
+                });
+                redisTools.saveToCache(CACHE_KEY, {result: apiRes.result, aggregations: apiRes.aggregations}, CACHE_EXPIRE);
+                apiRes.fromCache = false;
+                resolve({result: apiRes.result, aggregations: apiRes.aggregations});
+              }).catch((error) => {
+                reject(error);
+              });
+            }
+        });
+      } else {
+          reject("Couldn't process query.")
+      }
+      });
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your name needs to be translated, use this function getName().
+     * @returns the tab name translated
+     */
+
+    getName: function (i18n) {
+        const _ = i18n._;
+        return _("iot_pollution", "iot_pollution");
+    },
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * Otherwise, if your name doesn't need to be translated, use this attribute.
+     */
+
+    name: "iot_pollution",
+
+    /**
+     * (OPTIONAL/NEEDED)
+     * If your keyword needs to be translated, use this function getKeyword().
+     * @returns keyword translated
+     */
+    getKeyword: function (i18n) {
+        const _ = i18n._;
+        return _("datapollution", "iot_pollution");
```

Comment:
You set a translation key without its translation in po files.
You can add this in en.po
`msgctxt "iot_pollution"`
`msgid "datapollution"`
`msgstr "datapollution"`